cfme/middleware/deployment.py

```diff
@@ -0,0 +1,12 @@
+from cfme.fixtures import pytest_selenium as sel
+
+
+class Deployment(object):
+
+    def __init__(self, name):
+        self.name = name
+        self.provider = 'Middleware'
```

Comment:
It would be helpful if in model examples could be provided. The same for a test case example, given that some current test cases apparently are using deprecated fixture annotations, which we do not want to model after.